tools/Docs/az-ps-latest.csv

```diff
@@ -0,0 +1,123 @@
+pac90,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.PrivateDns,1.0.3
+pac91,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.ProviderHub,0.1.0
+pac92,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.RecoveryServices,4.3.0
+pac93,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.RedisCache,1.4.0
+pac94,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.RedisEnterpriseCache,1.0.0
+pac95,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Relay,1.0.3
+pac96,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Reservations,0.9.0
+pac97,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.ResourceGraph,0.11.0
+pac98,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.ResourceMover,1.0.0
+pac99,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Resources,4.2.0
+pac100,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Search,0.8.0
+pac101,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Security,1.0.0
+pac102,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.SecurityInsights,1.0.0
+pac103,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.ServiceBus,1.5.0
+pac104,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.ServiceFabric,3.0.0
+pac105,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.SignalR,1.3.0
+pac106,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.SpringCloud,0.2.0
+pac107,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Sql,3.4.0
+pac108,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.SqlVirtualMachine,1.1.0
+pac109,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.StackEdge,0.1.0
+pac110,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.StackHCI,0.8.0
+pac111,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Storage,3.9.0
+pac112,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.StorageSync,1.5.0
+pac113,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.StreamAnalytics,2.0.0
+pac114,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Subscription,0.8.0
+pac115,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Support,1.0.0
+pac116,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Synapse,0.13.0
+pac117,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.TimeSeriesInsights,0.2.0
+pac118,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.TrafficManager,1.0.4
+pac119,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.VMware,0.2.0
+pac120,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Websites,2.8.0
+pac121,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.WindowsIotServices,0.1.0
+
```

Comment:
Just to be careful, is it safe to have an empty line at the end?

Reply:
An empty line doesn't matter. This csv has been verified by docs team and was used to generated docs of az-ps-6, which is published now.